.github/e2e-tests.yml

```diff
@@ -0,0 +1,128 @@
+# This file specifies the GitHub runner for each E2E test and is utilized by all E2E CI workflows.
+#
+# Each entry in this file includes the following:
+# - The GitHub runner (runs-on field) that will execute tests.
+# - The tests that will be run by the runner.
+# - The workflows (e.g., Run PR E2E Tests, Run Nightly E2E Tests) that should trigger these tests.
+#
+runner-test-matrix:
+
+  # Example of 1 runner for all tests in integration-tests/smoke/ocr_test.go
+  - id: smoke/ocr_test.go:* 
+    path: integration-tests/smoke/ocr_test.go
+    test-type: docker
+    runs-on: ubuntu-latest
+    workflows:
+      - Run PR E2E Tests
+      - Run Nightly E2E Tests
+    test-cmd: cd integration-tests/ && go test smoke/ocr_test.go -timeout 30m -count=1 -json
+    pyroscope-env: ci-smoke-ocr-evm-simulated-nightly
+
+  # Example of 2 separate runners for the same test file but different tests. Can be used if tests if are too heavy to run on the same runner
+  - id: smoke/ocr2_test.go:^TestOCRv2Request$
+    path: integration-tests/smoke/ocr2_test.go
+    test-type: docker
+    runs-on: ubuntu-latest
+    workflows:
+      - Run PR E2E Tests
+      - Run Nightly E2E Tests
+    test-cmd: cd integration-tests/ && go test smoke/ocr2_test.go -test.run ^TestOCRv2Request$ -test.parallel=1 -timeout 30m -count=1 -json
+    pyroscope-env: ci-smoke-ocr2-evm-simulated-nightly
+
+  - id: smoke/ocr2_test.go:^TestOCRv2Basic$
+    path: integration-tests/smoke/ocr2_test.go
+    test-type: docker
+    runs-on: ubuntu-latest
+    workflows:
+      - Run PR E2E Tests
+      - Run Nightly E2E Tests
+    test-cmd: cd integration-tests/ && go test smoke/ocr2_test.go -test.run ^TestOCRv2Basic$ -test.parallel=1 -timeout 30m -count=1 -json
+    pyroscope-env: ci-smoke-ocr2-evm-simulated-nightly
+
+  # Automation upgrade tests
```

Comment:
Are we only running automation and OCR tests as a part of this? Are others being added on later?

Reply:
@kalverra we will add all of the E2E tests in the next PRs

Reply:
@lukaszcl I understand we will also remove all the other actions we have currently that deal with base64 config once we have migrated all workflows to the new way of life? do we have a ticket remind us to do so?